resources/views/auth/login.blade.php

```diff
@@ -3,19 +3,25 @@
 @section('content')
 <div class="container">
     <div class="row">
-        <div class="col-6 col-lg-6 col-md-12 col-sm-12">
+        <div class="col-sm-5">
+            <div class ="form-box">
+                <div class="form-top">
+                    <div class="form-top-left">
             <div class="panel panel-default">
                 <div class="panel-heading h1">Login</div>
-
+                    </div>
+                    
+                </div>
+                <div class="form-bottom">
                 <div class="panel-body">
                     <form class="form-horizontal" method="POST" action="{{ route('login') }}">
                         {{ csrf_field() }}
 
                         <div class="form-group{{ $errors->has('email') ? ' has-error' : '' }}">
-                            <label for="email" class="col-md-4 control-label">E-Mail Address</label>
+                            
 
                             <div class="col-md-6">
```

Comment:
Rather than having each input in a separate `<div class="col-md-6">,` can you put the whole form in one? For example
**Login**
`<div class="col-md-6">`
Login code
`</div>`

**Register**
`<div class="col-md-6">`
register code
`</div>`

Reply:
![image](https://user-images.githubusercontent.com/31779550/31596355-fc29e5d0-b20f-11e7-8c3d-ee68c328c3ec.png)

Reply:
This is what occurs when i tried doing that

Reply:
Oh that's odd